src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1156,4 +1157,18 @@ function ($a) {
             ->end()
         ;
     }
+
+    private function addRobotsIndexSection(ArrayNodeDefinition $rootNode)
+    {
+        $rootNode
+            ->children()
+                ->booleanNode('disallow_search_engine_index')
+                    ->info('Ask search engine to avoid indexing your application. Enabled by default when debug is enabled.')
```

Comment:
search engine**s** to not index the application?

Reply:
Fixed